### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by OceanOak**
Current state:

https://github.com/darklang/darklang.com/assets/87861924/077f96f5-5093-4dc4-9777-d99925d00b61

https://github.com/darklang/darklang.com/assets/87861924/6d73e244-e55a-4eda-aca9-f08137476eac

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,336 +6,362 @@
...
-              <li><a href="https://docs.darklang.com/changelog" target="_blank">Changelog</a></li>
+            <li class="text-purple"><span class="text-dark-charcoal">Works well with GitHub Copilot, VS Code, etc.</span></li>
           </ul>
-      </div>
+        </li>
+
+        <li id="section5" class="font-bold text-xl mt-8 scroll-mt-24">
+          Open source
+          <ul class=" list-outside text-lg font-normal m-4">
+            <li>In 2023, we expect to full open source darklang, which is currently source available</li>
```
Comment (StachuDotNet) [inline public/index.html:221]:
```suggestion
            <li>In 2023, we expect to fully open source Darklang, which is currently source available</li>
```

File: public/index.html [hunk 2]
```diff
@@ -6,336 +6,362 @@
...
-    <!--Features-->
-      <section class="relative pt-24">
-        <div class="absolute top-0 right-0">
-          <img src="./img/grid.png" alt="grid">
-        </div>
-        <div class="container text-white mx-auto text-center lg:text-3xl xs:text-xl m-10">
-          <p class=""><span class="text-purple">Darklang </span>is a platform and language to easily develop backend cloud applications. You can build tiny applications to connect two services, or large scale applications with tens of thousands of users. Code is written using AI and instantly and safely deployed on our hosted platform.</p>
-        </div>
+            <li class="text-purple"><span class="text-dark-charcoal">Designed to enable generated programs. Run partial or incorrect programs, correct incorrect Exposed language
+            internals, parser, error messages.</span></li>
```
Comment (StachuDotNet) [inline public/index.html:204]:
```suggestion
            <li class="text-purple"><span class="text-dark-charcoal">Designed to enable generated programs. Run partial or incorrect programs</span></li>
            <li class="text-purple"><span class="text-dark-charcoal">Exposed language
            internals, parser, error messages.</span></li>
```

Comment (StachuDotNet) [reply public/index.html:204]: the copy here was a bit off - this feels like a minor improvement

**Review by pbiggar — APPROVED** [review]
Looks good! I haven't looked more closely, but I'd propose we merge and iterate.

**Pull request merged**